tests/__mocks__/@myparcel/js-sdk/dist/data/delivery-options/getDeliveryOptionsEntry.js

```diff
@@ -1,3 +1,45 @@
+const morningEveningShipmentOptions = [
+  {
+    name: 'large_format',
+    schema: {
+      type: 'boolean',
+      enum: [
+        true,
+        false,
+      ],
+    },
+  },
+  {
+    name: 'only_recipient',
+    schema: {
+      type: 'boolean',
+      enum: [
+        true,
+      ],
```

Comment:
hier geen false?

Reply:
Sorry dat het niet duidelijk is, dit hele bestand is een api response. Er zijn een paar kleine dingetjes modulair gemaakt zoals die datum maar dit is hoe het uit de api komt. Overigens betekent dit dat hij geforceerd true is, dus bij ochtend/avond is dit uitzetten gewoon geen optie, dus dan krijg je selected+disabled :)